.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# Global rule:
+*           @AdrianGlasnapp-TomTom @joserojas-tomtom @MichalKaczmarek-TomTom @LukaszWitkowski-tomtom
```

Comment:
can you please add myself, Ioana, Subhransu Sekhar Sahoo and @iuliaromaniuc-tomtom?